docs/api_walkthrough.rst

```diff
@@ -305,6 +305,40 @@ If the balances of the tokens involved for Alice and Bob are now checked they sh
 
 
 
-Interacting with the Raiden echo node
+Interacting with the Raiden Echo Node
 =====================================
-TODO once the echo node is ready
+For easy testing of Raiden, there is a specialized Raiden node running, the "Raiden Echo Node". The Echo Node responds to received transfers by sending a transfer back to the initiator. The echo transfer follows certain rules:
+
+- consecutive transfers with the same ``identifier`` and same ``amount`` are ignored
+- the ``echo_identifier`` of all echo transfers is ``identifier + echo_amount``
+- transfers with an ``amount`` divisable by ``3`` will be answered with an echo transfer of ``echo_amount = amount - 1``
+- transfers with an ``amount = 7`` are special lottery transfers. They will go to a lottery pool. After the Echo Node has received seven lottery transfers, it will chose a winner that receives an echo transfer with ``echo_amount = 49`` and the pool is reset. To query the current number of tickets in the pool, a participant can send another transfer with ``amount = 7`` -- if the participant already takes part in the current draw, the Echo Node will respond with a transfer with ``echo_amount = lottery_pool_size``, otherwise it will enter the pool.
+- for a transfer with any other ``amount`` it returns ``echo_amount = amount``
+
+
+The address of the Echo Node is ``0x02f4b6bc65561a792836212ebc54434db0ab759a`` and it is connected to the Raiden Testnet Token (RTT) with the address ``0x0f114a1e9db192502e7856309cc899952b3db1ed``. The RTT token contract is verified and can be seen in `etherscan <https://ropsten.etherscan.io/address/0x0f114a1e9db192502e7856309cc899952b3db1ed#code>`_.To interact with the Echo Node, first :ref:`join the RTT network <joining-existing-token-network>`.
+
+You can obtain RTT tokens by calling the ``mint()`` function of the token. In javascript you can load the RTT token contract and call mint as such:::
```

Comment:
Maybe use "in a JavaScript Web3 console" instead of just "in javascript"?

Reply:
Why? Same thing can be used in npm javascript or any other way you may end up loading the javascript to interact with the contract.